app/views/admin/resources/new.html.haml

```diff
@@ -1 +1 @@
-= render 'form'
+= render 'form', resource: @resource
```

Comment:
It looks that way. But in any case, please check if everything is working properly prior to sending a PR.